.ci_support/osx_python3.7.yaml

```diff
@@ -13,31 +13,31 @@ cxx_compiler:
 expat:
 - '2.2'
 gdal:
-- '2.2'
+- '2.3'
```

Comment:
@ocefpaf where/how is this getting set?

Reply:
These are the pins used in the build. Let me send a PR to conda-forge-pinning, where this comes from, adding an optional pinning for `vc9`, that means we may not build things for `vc9` on the new `gcc7`, but they can still work after a re-render. Hold on...